src/cpp/r/RJson.cpp

```diff
@@ -162,7 +162,7 @@ Error jsonValueFromVectorElement(SEXP vectorSEXP,
                }
                case UINT32:
                {
-                  if ((value >= 0) && (value <= UINT32_MAX))
```

Comment:
Why remove this check?

Reply:
It's getting a compiler warning on the mac because int is not of the same precision as the uint32 it's being compared to.  From my reading of the code, a signed int is always less than uint32_max so it's not necessary right?  If R supported unsigned ints, we'd have a different better way to fix it!